lib/setup.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python3
+
+from setuptools import setup, find_packages
+
+setup(
+    name='renewablepotentialslib',
+    version='0.1.0', # additionally defined in __init__.py
+    description='Library code of the solar-and-wind-potentials workflow.',
+    maintainer='calliope-project',
+    maintainer_email='bryn.pickering@usys.ethz.ch',
```

Comment:
Slightly delicate matter, but would you mind adding my email address here? What you definitely should do is add your name to the LICENSE.md file (and update the year while you are there).

Reply:
I suspect that a better longterm solution would be an email address that is not associated with an individual nor an institute. Even if linked only to you, it would be easier if it isn't linked to your ETH email.

Reply:
I agree.